main/java/mjc/Euro.java

```diff
@@ -0,0 +1,59 @@
+package main.java.mjc;
+
+import java.util.Locale;
+
+public class Euro implements Comparable<Euro> {
+    private final int value;
+
+    public Euro(int cents) {
+        this.value = cents;
+    }
+
+    public Euro(int euros, int cents) {
+        this.value = euros * 100 + cents;
+    }
+
+    public Euro(Euro euro) {
+        this.value = euro.value;
+    }
+
+    public Euro add(Euro euro) {
+        return new Euro(this.value + euro.value);
+    }
+
+    public Euro sub(Euro euro) {
+        return new Euro(this.value - euro.value);
+    }
+
+    public Euro mul(int k) {
+        return new Euro(this.value * k);
+    }
+
+    @Override
+    public String toString() {
+        String formattedValue = String.format(Locale.ROOT, "%.02f", value * 1.);

```

Comment:
`value * 1.` isn't exact operation. 
and i'm not sure about rounding. 
separate the `value` into two parts by integer operations and join results with a format string.